README_LINUX.md

```diff
@@ -108,7 +108,17 @@ Unfortunately, the Qt installer does not allow you to deselect the multi-gigabyt
 Building
 --------
 
-### Step 1: Make a build directory
+### Step 1: Clone the repository
+
+SuperCollider is hosted on Github: https://github.com/SuperCollider/SuperCollider
+
+First, clone the repository with git:
+
+    git clone --recursive https://github.com/SuperCollider/SuperCollider.git
```

Comment:
I know I'm being picky but you shouldn't use uppercase characters in URLs

Reply:
I hadn't noticed that. Thanks!